rest/__tests__/service/networkNodeService.test.js

```diff
@@ -576,6 +576,130 @@ describe('NetworkNodeService.getNetworkNodes tests node filter', () => {
   });
 });
 
+describe('NetworkNodeService node_account_id override logic', () => {
+  const nodeAccount1000 = EntityId.parseString('0.0.1000');
+  const nodeAccount1001 = EntityId.parseString('0.0.1001');
```

Comment:
do not include `shard.realm.` so the test will also work for non-zero shard / realm

```suggestion
  const nodeAccount1000 = EntityId.parseString('1000');
  const nodeAccount1001 = EntityId.parseString('1001');
```